### ISSUE TITLE
No favicons in Full Screen mode

### ISSUE THREAD

**Issue by tollfree** [hetima/SafariStand#51]
The favicons disappear after entering Full Screen mode.
v9.0.217Yocemite, OSX 10.11.3, Safari 9.0.3.

**Comment by noizo** [comment]
Confirmed on El Capitan

**Comment by piratevolikov** [comment]
Double-confirmed on El Capitan latest version.
Installed SafariStand for Yosemite using "csrutil disable/enable" on installing SIMBL.
No other extensions are installed except for AdBlock

**Comment by mtsvik** [comment]
Same problem on El Capitan

**Comment by tollfree** [comment]
[Here](https://github.com/hetima/SafariStand/pull/47#issuecomment-217422826) is a fixed bundle.

**Comment by piratevolikov** [comment]
Replaced the bundle in the Libraries/.../Plugins/, restarted Safari. Nothing's changed

**Comment by mtsvik** [comment]
@tollfree Didn't help too.

**Comment by tollfree** [comment]
Use ~/Library/Application Support/SIMBL/Plugins.

**Comment by mtsvik** [comment]
@tollfree yep, I did just that

**Comment by piratevolikov** [comment]
@tollfree yes, I also did the same. Although the SIMBL folder is located in %username%/Library/Application Support, not in ~/Library/Application Support (not in global Library folder)

**Comment by ylluminate** [comment]
Maybe @w0lfschild should chime in here. With the lack of support from the original developer and SIMBL being such a valuable tool, we need to bring some more attention to this.
